doc/blobfuse2.md

```diff
@@ -0,0 +1,29 @@
+## blobfuse2
```

Comment:
Should we add this to the release pipeline so it automatically does this for us?

Reply:
Seems unnecessary. These parameters are not going to be changing on each release